extensions/wikia/Search/templates/WikiaSearch_CrossWiki_index.php

```diff
@@ -10,7 +10,7 @@
 						<?= wfMsg('wikiasearch2-results-for', '<strong>' . $query . '</strong>'); ?>
 					<?php endif; ?>
 					<?php if (isset($hub) && $hub) : ?>
-						<?= wfMsg('wikiasearch2-onhub', $hub) ?>
+						<?= wfMessage( 'wikiasearch2-onhub', Sanitizer::stripAllTags( $hub ) )->text() ?>
```

Comment:
Yeah, I changed it to the text and should other way around - thanks for noticing!